lib/build.js

```diff
@@ -60,6 +60,7 @@ module.exports = {
     this.stdout = new Buffer(0);
     this.stderr = new Buffer(0);
     atom.commands.add('atom-workspace', 'build:trigger', this.build.bind(this));
+    atom.commands.add('atom-workspace', 'build:toggle-view', this.toggleView.bind(this));
```

Comment:
Oh, didn't know it's ok to register commands in `build-view.js` (I guess that's because there were none).

But it feels kind of wrong to register commands in the build view. I mean, it looks like we can create multiple instances of BuildView but Atom commands seem kind of like global things, wouldn't that create problems?